dev/TabView/TabView.xaml

```diff
@@ -438,7 +438,7 @@
                                     <VisualState.Setters>
                                         <Setter Target="TabContainer.Background" Value="{ThemeResource TabViewItemHeaderBackgroundSelected}" />
                                         <Setter Target="ContentPresenter.Foreground" Value="{ThemeResource TabViewItemHeaderForegroundPressed}" />
```

Comment:
In Selected, foreground is always the same. Would make more sense to not use TabViewItemHeaderForegroundPointerOver and Pressed for selected states but rather have a TabViewItemHeaderForegroundSelected brush, which is used for both icon and header text.